dataland-frontend/src/components/forms/parts/elements/derived/ActivityTree.ts

```diff
@@ -0,0 +1,970 @@
+import { type TreeNode } from "primevue/tree";
+
+export const activityTree: Array<TreeNode> = [
+  {
+    name: "Forestry",
+    key: "1",
+    sector: "Forestry",
+    type: "header",
+    children: [
+      {
+        name: "Afforestation",
+        value: "Afforestation",
+        key: "1.1",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Rehabilitation and restoration of forests, including reforestation and natural forest regeneration after an extreme event",
+        value:
+          "RehabilitationAndRestorationOfForestsIncludingReforestationAndNaturalForestRegenerationAfterAnExtremeEvent",
+        key: "1.2",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Forest management",
+        value: "ForestManagement",
+        key: "1.3",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Conservation forestry",
+        value: "ConservationForestry",
+        key: "1.4",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+    ],
+  },
+  {
+    name: "Environmental protection and restoration activities",
+    key: "2",
+    sector: "Environmental protection and restoration activities",
+    type: "header",
+    children: [
+      {
+        name: "Restoration of wetlands",
+        value: "RestorationOfWetlands",
+        key: "2.1",
+        sector: "Environmental protection and restoration activities",
+        type: "child",
+        nace_codes: null,
```

Comment:
nace_codes: `null` ==> When I select this in the UI, I can't select a Nace code (because there are none), but I can't submit the form because Nace-Code is required.

Reply:
fixed

Reply:
When I go on edit of "Abott, Nikolaus and Thiel" This gem turns up:
![grafik](https://github.com/d-fine/Dataland/assets/71646830/de41c8e5-f4c2-4079-b2ad-ad4ab7bd78d4)

Reply:
When no NACE code for an Activity is available the field in the form shows null and upload is possible. However, it would be nicer to have something like "No NACE code" available displayed instead. Is that possible?

Reply:
> When I go on edit of "Abott, Nikolaus and Thiel" This gem turns up: ![grafik](https://user-images.githubusercontent.com/71646830/267305922-de41c8e5-f4c2-4079-b2ad-ad4ab7bd78d4.png)

This null's are displayed because fake fakefixtures includes strange NACE codes like (F, A, D, ...) if you choose the NACE code assigned to the activity (02 - Forestry and logging), this situation does not occur. Do we want to handle incorrect data for NACE codes for activists?

Reply:
> 

![image](https://github.com/d-fine/Dataland/assets/96523011/987dc520-efd3-4981-99ba-19a1cd04cf29)